src/test/java/apoc/export/ExportS3PerformanceTest.java

```diff
@@ -0,0 +1,116 @@
+package apoc.export;
+
+import apoc.export.csv.ExportCSV;
+import apoc.graph.Graphs;
+import apoc.util.TestUtil;
+import apoc.util.s3.S3Aws;
+import apoc.util.s3.S3Params;
+import apoc.util.s3.S3ParamsExtractor;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.factory.GraphDatabaseSettings;
+import org.neo4j.test.TestGraphDatabaseFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Optional;
+
+import static apoc.util.MapUtil.map;
+import static junit.framework.TestCase.assertTrue;
+
+@Ignore("To use this test, you need to set the S3 bucket and region to a valid endpoint " +
```

Comment:
Instead of this ignore we can leverage Loadstack in combination with Testcontainers:

https://www.testcontainers.org/modules/localstack/

can you please create a base class with it and extend each one that you created?